### ISSUE TITLE
--live-reload not passed on to `nanoc view` when invoked as `nanoc live`

### ISSUE THREAD

**Issue by moll** [guard/guard-nanoc#38]
Hey,

From the output of `--help`, I believe `nanoc live` intended to pass its arguments to `nanoc view`, however for `--live-reload` it still throws an error about an illegal option. It's probably more future proof to pass unrecognized args all through as I'd probably forget to update `nanoc live`'s opt parser as well if I were improving `nanoc view`. :P

**Comment by ddfreyne** [comment]
Hi,

`nanoc live` doesn’t pass options to `nanoc view`.  It always has live-reloading enabled, so there’s no need to pass `--live-reload`.

Am I correct when I say that your confusion is caused by the following:

```
DESCRIPTION
    Starts the live recompiler along with the static web server. [snip]
```

If so, I think a better description might be

```
DESCRIPTION
    Starts the live recompiler, along with a web server with live-reloading enabled. [snip]
```

What do you think?

**Comment by ddfreyne** [comment]
Oh, my bad — I am confusing the `nanoc live` command provided by guard-nanoc and the `nanoc live` command provided by the `nanoc-live` gem (pre-release only for now).

Options _are_ meant to be passed through, so this is indeed incorrect.

**Comment by ddfreyne** [comment]
This is fixed in 2.1.3, just released!

**Comment by moll** [comment]
Great! Thank you!
